api/src/paths/projects.unit.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
```

Comment:
Can we move these tests into the other test file? We only need to maintain 1 test file (per code file), that can hold all of the tests.

Reply:
yes